Content.Server/GameTicking/Rules/ZombieRuleSystem.cs

```diff
@@ -192,29 +194,27 @@ private void OnZombifySelf(EntityUid uid, ZombifyOnDeathComponent component, Zom
         _action.RemoveAction(uid, action);
     }
 
-    private float GetInfectedPercentage(out List<EntityUid> livingHumans)
+    private float GetInfectedFraction()
     {
-        var allPlayers = EntityQuery<HumanoidAppearanceComponent, MobStateComponent>(true);
-        var allZombers = GetEntityQuery<ZombieComponent>();
+        var players = EntityQuery<HumanoidAppearanceComponent>(true);
+        var zombers = EntityQuery<HumanoidAppearanceComponent, ZombieComponent>(true);
```

Comment:
why query HumanoidAppearanceComponent, its unnecesary and it looks like you only need ZombieComponent here, also take the queries as parameters since you repeat them for GetHealthyHumans so you can feed the queries to both methods if you make em on OnRoundEndText handler

Reply:
HumanoidAppearance is here because we only count humanoids

Reply:
also, its only run once when round ends, it shouldn't be a big performance hit for not reusing queries (they are different even)